src/components/Player/Header/PlayerHeader.jsx

```diff
@@ -52,43 +46,71 @@ const PlayerName = ({ playerName, playerId, picture, registered, loading, error,
       height: 18,
     };
 
+    let avatarStyle = {
+      marginLeft: width > HEADER_MD_BREAK ? 30 : 0,
+      marginRight: width > HEADER_SM_BREAK ? 30 : 0,
+    };
+
+    if (width <= HEADER_MD_BREAK) {
+      badgeStyle = {
+        ...badgeStyle,
+        marginLeft: -1 * (LARGE_IMAGE_SIZE / 2) * 0.75,
+      };
+    }
+    if (width <= HEADER_SM_BREAK) {
+      avatarStyle = {
+        ...avatarStyle,
+        marginLeft: LARGE_IMAGE_SIZE * 0.75,
+      };
+      badgeStyle = {
+        ...badgeStyle,
+        marginLeft: LARGE_IMAGE_SIZE * 0.5,
+      };
+    }
+
     return (
       <div className={styles.container}>
-        <div>
-          <CardHeader
-            style={{ padding: 0 }}
-            avatar={
-              <Badge
-                badgeContent={getRegistrationBadge(registered)}
-                badgeStyle={badgeStyle}
-                style={{
-                  margin: 0,
-                  padding: 0,
-                }}
-              >
-                <Avatar
-                  src={picture}
-                  style={{
-                    marginLeft: width > HEADER_BREAK_SIZE ? 30 : 0,
-                    marginRight: 30,
-                  }}
-                  size={width > HEADER_BREAK_SIZE ? 124 : 62}
-                  className={styles.oreviewAvatar}
-                />
-              </Badge>
-            }
-            title={getTitle(playerName, playerId, width)}
-            titleStyle={{ fontSize: 28, marginTop: 6 }}
-            subtitle={width > HEADER_BREAK_SIZE ?
-              <PlayerStats playerId={playerId} /> :
-                <PlayerBadges playerId={playerId} />
-            }
-          />
-          {width <= HEADER_BREAK_SIZE && <PlayerStats playerId={playerId} />}
-        </div>
-        <div className={styles.playerButtonsContainer}>
-          <PlayerButtons />
+        <div className={styles.topContainer}>
+          <Badge
+            badgeContent={getRegistrationBadge(registered)}
+            badgeStyle={badgeStyle}
+            style={{
+              margin: 0,
+              padding: 0,
+            }}
+          >
+            <Avatar
+              src={picture}
+              style={avatarStyle}
+              size={width > HEADER_MD_BREAK || width <= HEADER_SM_BREAK ? LARGE_IMAGE_SIZE : LARGE_IMAGE_SIZE / 2}
```

Comment:
Ok so no matter how we do this, the current implemention is the cleanest. It doesn't make sense to not pass the size of the avatar in js, but explicitly set breakpoints in the css (from a readability point of view). I did get it to work by using css, but it's messier.